src/anyp/PortCfg.h

```diff
@@ -36,15 +36,17 @@ class PortCfg : public CodeContext
     virtual ScopedId codeContextGist() const override;
     virtual std::ostream &detailCodeContext(std::ostream &os) const override;
 
+    void print(std::ostream &) const;
+
     PortCfgPointer next;
 
+    /// actual or implied (by squid -a) squid.conf directive name
+    SBuf directiveName;
+    TrafficMode flags;  ///< flags indicating what type of traffic to expect via this port
```

Comment:
Just slightly better formatting and wording for this moved field:

```suggestion

    /// the kind of traffic this port expects
    TrafficMode flags;

```